context.cpp

```diff
@@ -321,6 +306,7 @@ namespace Sass {
 
   string Context::format_source_mapping_url(const string& file)
   {
+/*
```

Comment:
This is fairly unsafe. Use `#if 0` if you absolutely must comment out a block (but removal is better).